bin/instance/utils/convert-for-k8s.sh

```diff
@@ -421,7 +424,9 @@ LOCAL_CA_PASSWORD=local_ca_password
 LOCAL_CA_ALIAS=localca
 LOCAL_CA_FILENAME="local_ca/localca"
 NEW_ZWE_EXTERNAL_HOSTS=localhost
+NEW_ZWE_EXTERNAL_PORT=7445
 VERBOSE_MODE=
+SILENT_MODE=1
```

Comment:
Feel like `SILENT_MODE` here is more about `VERBOSE_MODE`. To make the variable more meaningful,

- preset `SILENT_MODE=`,
- if `-q` is in parameter, set `SILENT_MODE=true`,
- check `if [ "${SILENT_MODE}" != "true" ]; then` to display messages.